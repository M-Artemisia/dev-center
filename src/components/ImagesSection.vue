<template>
  <div class="images-section">
    <div class="syn-bg-gray syn-flex">
      <slot></slot>
    </div>
  </div>
</template>

<script>

export default {
  name: 'ImagesSection',
}
</script>
