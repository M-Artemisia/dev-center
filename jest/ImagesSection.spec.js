import { mount } from '@vue/test-utils'
import ImagesSection from '@/components/ImagesSection'

describe('ImagesSection', () => {
  test('should fill slot with content provided', () => {
    const wrapper = mount(ImagesSection, {
      slots: {
        default: `<div class="image-popout-stub"></div>`
      }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
