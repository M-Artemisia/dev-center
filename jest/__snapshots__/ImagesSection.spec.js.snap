// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImagesSection should fill slot with content provided 1`] = `
<div class="images-section">
  <div class="syn-bg-gray syn-flex">
    <div class="image-popout-stub"></div>
  </div>
</div>
`;
